scripts/PrepManualTesting.ps1

```diff
@@ -43,30 +46,31 @@ if(!(Test-Path -Path $PackageDir)) {
     throw "Directory '$PackageDir' does not exist, specify -Build to generate"
 }
 
-$nupkgFiles = Get-ChildItem -Path $PackageDir -Filter *.nupkg
-if(!$nupkgFiles) {
-    throw "No NUPKG files found, specify -Build to generate"
-}
-
-$nonNuPkgFiles = Get-ChildItem -Path $PackageDir -Exclude *.nupkg
-if($nonNuPkgFiles) {
-    if(!$Force -and !$PSCmdlet.ShouldContinue("$($nonNuPkgFiles | ForEach-Object { $_.FullName } | Out-String)", "Delete files")) {
-        exit 0
+$PackageDir = (Resolve-Path -Path $PackageDir).ProviderPath
+if(!$LoadOnly) {
+    $nupkgFiles = Get-ChildItem -Path $PackageDir -Filter *.nupkg
+    if(!$nupkgFiles) {
+        throw "No NUPKG files found, specify -Build to generate"
     }
 
-    Write-Verbose "Removing non-NUPKG files..."
-    $nonNuPkgFiles | Remove-Item -Force -ErrorAction Stop -Recurse
-}
+    $nonNuPkgFiles = Get-ChildItem -Path $PackageDir -Exclude *.nupkg
```

Comment:
This doesn't match the casing of your previous variable. "nupkg" vs. "nuPkg". I think this should be completely lowercase.

Reply:
Doesn't matter on casing for PowerShell variables, nupkg is extension but name and typically listed in documentation is "NuGet Packages" or "NuPkg" for short